Controllers/Settings.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Leantime\Plugins\OmniSearch\Controllers;
+
+use Leantime\Core\Controller;
+
+/**
+ * Settings Controller for Sample Plugin
```

Comment:
This is not a sample plugin. It's the plugin to end all plugins!

Reply:
Removed the controller for now as it's unused. 
It can be re-added later when the time to epic configuration is!